tests/test_simple.py

```diff
@@ -288,3 +288,25 @@ def test_gpg_sig(self, html, expected_gpg_sig):
         archive_links = simple.parse_archive_links(html)
         assert len(archive_links) == 1
         assert archive_links[0].gpg_sig == expected_gpg_sig
+
+    @pytest.mark.parametrize(
+        "html,expected",
+        [
+            (
+                '<a href="spam-1.2.3-py3.none.any.whl" data-yanked>spam-1.2.3-py3.none.any.whl</a>',
+                True,
+            ),
+            (
+                '<a href="spam-1.2.3-py3.none.any.whl" data-yanked="oops!">spam-1.2.3-py3.none.any.whl</a>',
```

Comment:
Did you want to write another assert line (or separate test) that tests for the truthiness _and_ the content of the string?

Reply:
Ah I see. Fancy `mark.paramatrize` usage going on here!

Nevermind. Carry on.